### PR TITLE
Created xml for recommended users fragment.

### PR DESCRIPTION
**Pull request by OnlyAlf**

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (itzcoatl90) [inline .idea/codeStyles/Project.xml:1]: no

**Review by itzcoatl90 — DISMISSED** [review]
You know about the files

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -28,5 +28,6 @@
     <string name="tv_recyclerview_chat_user_item_group_chat_text">+5</string>
     <string name="tv_group_name">GROUP NAME</string>
     <string name="tv_group_description">GROUP DESCRIPTION</string>
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```
Comment (ecejmsr) [inline app/src/main/res/values/strings.xml]: Hello blank fragment needs removal

**Review by ecejmsr — CHANGES_REQUESTED** [review]

**Review by ecejmsr — APPROVED** [review]
Looks good.

**Pull request merged**